python-package/lightgbm/sklearn.py

```diff
@@ -349,19 +337,14 @@ def fit(self, X, y,
             params['num_class'] = self.n_classes_
         if hasattr(self, 'eval_at'):
             params['ndcg_eval_at'] = self.eval_at
-
         if self.fobj:
-            params["objective"] = "None"
-        else:
-            params["objective"] = self.objective
+            params.pop('objective', None)
```

Comment:
is there a `objective=None` option? ok, I will make it `params["objective"] = "None"`.